spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -345,5 +345,26 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
   def predictClass(dataset: RDD[Sample[T]]): RDD[Int] = {
     Predictor(this).predictClass(dataset)
   }
+
+  /**
+    * Set weight and bias for the module
+    * @param newWeights
+    * @return true: set weights successfully
+    *         false: set weights failed
+    */
+  def setWeights(newWeights: Array[Tensor[T]]): Boolean = {
```

Comment:
This is consistant with keras. The keras api "set_weights" is passing a list of numpy arrays which includes weight and bias.